_pages/ato/checklist.md

```diff
@@ -35,9 +35,9 @@ Many of these tasks can be done in parallel...this is just a suggestion of prior
 1. [ ] [Security scans](https://pages.18f.gov/before-you-ship/security/scanning/)
     * [ ] Set up [static analysis](https://pages.18f.gov/before-you-ship/security/static-analysis/) service
         * [ ] Add service badges to the README
-    * [ ] [Run OWASP ZAP](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
+    * [ ] [Perform automated vulnerability testing](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/)
```

Comment:
It is required for 18F, yes. Note that by "automated" I mean "using ZAP", not that it needs to use Compliance Viewer. Think that needs to be clarified?